package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ent-comp",
-  "version": "0.10.0",
+  "version": "0.10.1",
```

Comment:
This was done automatically by npm, presumably because there's a mismatch between this and package.json. Feel free to undo if it was for a reason!